roop/state.py

```diff
@@ -0,0 +1,64 @@
+import os
+
+from roop import utilities
+
+PROCESSED_PREFIX = '_'
+processed_frames_cnt = 0
+
+
+def is_exists(target_path: str) -> bool:
+    """
```

Comment:
Sorry but all the comments have to go... I reject to read them - if a method needs a paragraph to be explained, the method name is just bad.

Reply:
done